modMcf/src/org/aion/mcf/config/Cfg.java

```diff
@@ -136,11 +129,242 @@ public void setConsensus(CfgConsensus _consensus) {
         this.consensus = _consensus;
     }
 
+    /* ------------ execution path management ------------ */
```

Comment:
Supporting the old directory structure as well as the new one is definitely convenient and I'm sure users will appreciate it.  Is this something we should continue to support into the future though?

I think at some point (not in this PR) we should make a converter that converts the old structure to the new one so we don't have the burden of dealing with the two code paths.  The transition could go like this:

1. Make the script and release it
2. Add message in kernel so if it detects old structure, print a message saying they need to migrate it using the script at some point, because we'll stop supporting it 
3. Wait a while, deprecate the code that supports it